tests/phpunit/tests/media.php

```diff
@@ -3462,6 +3457,161 @@ public function data_wp_get_attachment_image_decoding_attr() {
 		);
 	}
 
+	/**
+	 * Tests that the `wp_get_loading_optimization_attributes()` function should add
+	 * the 'decoding' attribute.
+	 *
+	 * @ticket 58892
+	 *
+	 * @dataProvider data_add_decoding_attr
+	 *
+	 * @param string $tag_name The tag name.
+	 * @param string $decoding The value for the 'decoding' attribute. 'no value' for default.
```

Comment:
'no value' is a weird way to indicate a default :) I think you can simply use `null` instead. Even if not, you don't use 'no value' anywhere, and the function does not enforce it, so it's not worth documenting it.